netlify/util/dbUtil.js

```diff
@@ -0,0 +1,17 @@
+import { MongoClient } from 'mongodb';
+
+let cachedDb = null;
+
+export const connectToDatabase = async (uri) => {
+  // we can cache the access to our database to speed things up a bit
+  // (this is the only thing that is safe to cache here)
+  if (cachedDb) return cachedDb;
+
+  const client = await MongoClient.connect(uri, {
```

Comment:
TODO check what should be cached. Probably connections should be cached instead of database.